### ISSUE TITLE
Update requirements to avoid breaking clashes

### ISSUE THREAD

**Issue by Jack-Vines** [nestauk/dap_aria_mapping#31]
`pip install .` is failing due to package clashes:
```
ERROR: Cannot install boto3==1.24.93 and dap-aria-mapping because these package versions have conflicting dependencies.

The conflict is caused by:
    aiobotocore 2.4.1 depends on botocore<1.27.60 and >=1.27.59
    boto3 1.24.93 depends on botocore<1.28.0 and >=1.27.93

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflict
```
